backend/app/exporters/models/marc21.rb

```diff
@@ -149,8 +148,38 @@ def handle_id(*ids)
   end
 
 
-  def handle_title(title)
-    df('245', '1', '0').with_sfs(['a', title])
+  def handle_title(title, linked_agents, dates)
+    creator = linked_agents.find{|a| a['role'] == 'creator'}
+    date_codes = []
+
+    # process dates first, if defined. 
+    unless dates.empty?
+      dates = [["single", "inclusive", "range"], ["bulk"]].map {|types|
+        dates.find {|date| types.include? date['date_type'] }
+      }.compact
+
+      dates.each do |date|
+        code, val = nil
+        code = date['date_type'] == 'bulk' ? 'g' : 'f'
+        if date['expression']
+          val = date['expression']
+        elsif date['end']
```

Comment:
@avatar382 While this may result in the same outcome as what was spec'd out, to be more explicit about what is going on here, the elsif statement should check the value of the date['date_type'] instead of date['end'].

Reply:
After talking about this, it is implemented as needed.